Lanchonete/Lanchonete/ Features/Confirmation/Presentation/ConfirmationViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ConfirmationViewController.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 1/25/21.
+//
+
+import UIKit
+
+final class ConfirmationViewController: UIViewController {
+    
+    var presenter: ConfirmationViewPresenter?
+    
+    @IBOutlet weak var sandwichView: UIImageView!
+    
+    @IBOutlet weak var sandwichTitle: UILabel!
+    @IBOutlet weak var sandwichIngredientList: UILabel!
+    @IBOutlet weak var sandwichImage: UIImageView!
+    @IBOutlet weak var countLabel: UILabel!
+    
+    var sandwichType: SandwichType?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        sandwichTitle.text = presenter?.getSandwichName()
+        sandwichIngredientList.text = presenter?.getIngredientList()
+        sandwichImage.image = presenter?.getSandwichImage()
+    }
+    
+    @IBAction func plusButtonTapped(_ sender: UIButton) {
+        countLabel.text = presenter?.incrementQuantity()
```

Comment:
There are two things being done in here. I would suggest you to only call the `presenter?.incrementQuantity()` and let the view to be updated by another flow.
For example, the presenter, after incrementing the quantity, calls a method of the view like `view?.display(quantity: quantity)`

Reply:
the increment is already being tested in ConfirmationViewPresenterTests 😄